src/index.js

```diff
@@ -1,6 +1,29 @@
 import './style.css';
-import { createItems, tasks } from './createItems.js';
+import { createItems } from './createItems.js';
+import { addTask } from './addItem.js';
+import {
+  getTasks, saveTasks, clearCompletedTasks,
+} from './taskManager.js';
 
 document.addEventListener('DOMContentLoaded', () => {
-  createItems(tasks);
+  getTasks();
+  createItems();
+});
+
+const clearButton = document.querySelector('.clear-items button');
+
+clearButton.addEventListener('click', () => {
+  clearCompletedTasks();
+  saveTasks();
+  createItems();
+});
+
+const addButton = document.querySelector('.add-task button');
+const inputField = document.getElementById('add-tasks');
+
+addButton.addEventListener('click', addTask);
+inputField.addEventListener('keydown', (event) => {
+  if (event.key === 'Enter') {
+    addTask();
+  }
```

Comment:
> Implement a function for editing task descriptions.
- [ ] From the project requirement, you are meant to also implement the `task editing` feature in this milestone. I'll suggest you implement this in your project. :+1: :+1: